apps/src/userHeaderStatsigReporter/userHeaderStatsigReporter.js

```diff
@@ -0,0 +1,52 @@
+import statsigReporter from '../lib/util/StatsigReporter';
+
+const isInCreateAccountButtonExperiment = statsigReporter.getIsInExperiment(
+  'create_account_button',
+  'showCreateAccountButton',
+  false
+);
+
+export function showCreateAccountButton() {
+  console.log(isInCreateAccountButtonExperiment);
```

Comment:
nit: remove debug log

Reply:
I left this here in case we want to do a bug bash 😁 It makes it easier to see which bucket you're in.